src/components/Layout/NavBar/NavMenu.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="nav-menu">
+    <div v-for="(menuItem, index) in menuData" :key="index" class="menu-item">
```

Comment:
i think the menu-item should be a child component in itself, and utilize props to pass in the data it needs to display and function properly, while reducing the need to make changes in that file going forward. This makes the menu-item as reusable as possible. and having them as true components will help with layout and vue devtooling